src/client/components/household-information/FinancialDisclosureSection.jsx

```diff
@@ -36,7 +24,9 @@ class FinancialDisclosureSection extends React.Component {
             <td>{`${this.props.data.provideFinancialInfo ? 'Yes' : 'No'}`}</td>
           </tr>
           <tr>
-            <td>I understand VA is not enrolling new applicants who decline to provide their financial information:</td>
+            <td>I understand VA is not enrolling new applicants who decline to provide
+            their financial information, when that financial information is the only way
```

Comment:
I might change this to: `when that financial information may be the only way` instead of `is the only way`.